### PR TITLE
dist/prometheus: add prometheus config and rules

### LINKED ISSUE #293: Prometheus records wrong address families for UDP/HTTP announces and UDP connects
We don't set the `AddressFamily`, so it always defaults to `IPv4` for announces.

### PR DESCRIPTION
**Pull request by mrd0ll4r**
We might wanna fix #293 first, which in turn depends on #311, because right now address families are all over the place.

### CODE REVIEW

File: dist/prometheus/prometheus.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+rule_files:
+  - chihaya.rules
+
+# A scrape configuration containing exactly one endpoint to scrape:
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: 'your-job'
+    scrape_interval: 5s # optionally override the global scrape_interval
+    static_configs:
+      - targets: ['1.2.3.4:6881'] # provide the address of chihaya's prometheus endpoint
```
Comment (jzelinskie) [inline dist/prometheus/prometheus.yaml]: maybe we can make this `localhost` so it's a little more useable out of the box

File: dist/prometheus/prometheus.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  scrape_interval:     5s
+  evaluation_interval: 5s
+
+rule_files:
+  - chihaya.rules
+
+# A scrape configuration containing exactly one endpoint to scrape:
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: 'your-job'
```
Comment (jzelinskie) [inline dist/prometheus/prometheus.yaml]: name this local-chihaya or something

File: dist/prometheus/chihaya.rules [hunk 3]
```diff
@@ -0,0 +1,74 @@
+# Chihaya recording rules
```
Comment (jzelinskie) [inline dist/prometheus/chihaya.rules]: We should definitely have documentation on what rules are (how they are optional) etc...

**Comment by mrd0ll4r** [comment]
updated, ptal

**Comment by jzelinskie** [comment]
LGTM. Can you add another commit that moves `contrib/` into `dist/` as well. I think it makes more sense to consolidate with that convention.

**Comment by mrd0ll4r** [comment]
Will do, I can't merge this without you pressing that approve button though :)

**Pull request merged**